build/examples/agent-control-config-nr-otel-collector.yaml

```diff
@@ -1,15 +1,15 @@
-# fleet_control:
-#   endpoint: https://opamp.service.newrelic.com/v1/opamp
-#   signature_validation:
-#     public_key_server_url: https://publickeys.newrelic.com/r/blob-management/global/agentconfiguration/jwks.json
-#   headers:
-#     api-key: API_KEY_HERE
-#   fleet_id: FLEET_ID_HERE
-#   auth_config:
-#     token_url: PLACEHOLDER
-#     client_id: PLACEHOLDER
-#     provider: PLACEHOLDER
-#     private_key_path: PLACEHOLDER
+fleet_control:
```

Comment:
Do we still need the examples after adding the cli? Should we remove them?
Why did you only uncomment stuff in this example, but not the others?

Reply:
Great catch! The example should be untouched for now.

The examples will be removed but they can't  be removed until the recipe is updated. I was thinking following these steps:

1. Finish the implementation (this PR)
2. Include the new cli in the package (along with nr-auth cli and the examples)
3. Release it.
4. Update the recipe to use the new cli (and stop 
5. Clean everything up: stop packaging the nr-auth cli and remove the examples.